joomla/cassiopeia
# Create Sticky footer

Pull Request for Issue # .

### Summary of Changes
Allows for sticky footer
removes trailing space when sticky is not used.

### Testing Instructions
observe back-to-top only shows if your at the very bottom of the page
apply pr.
in template styles turn on "sticky footer"
run npm ci
observe back-to-top stays @ bottom of view port

### Expected result
![image](https://user-images.githubusercontent.com/1850089/94037896-7301ca00-fd8b-11ea-892c-8cad3b5b11af.png)

![image](https://user-images.githubusercontent.com/1850089/94037833-62515400-fd8b-11ea-9d51-ba291046bf2c.png)


### Actual result
![image](https://user-images.githubusercontent.com/1850089/94037976-890f8a80-fd8b-11ea-9cc9-f6b04e98b432.png)
![image](https://user-images.githubusercontent.com/1850089/94038005-90cf2f00-fd8b-11ea-8730-0af3e9a0c6bf.png)

### Documentation Changes Required
